src/net/peacefulcraft/cowbot/handlers/GameChatMessageHandler.java

```diff
@@ -8,32 +8,39 @@
 import net.md_5.bungee.api.chat.ComponentBuilder;
 import net.md_5.bungee.api.connection.ProxiedPlayer;
 import net.peacefulcraft.cowbot.CowBot;
+import net.peacefulcraft.cowbot.translation.DiscordToMinecraftFormattingTranslator;
 
 public class GameChatMessageHandler {
 
   public static void handle(Message message) {
     String author = message.getAuthor().get().getUsername();
     Member sender = message.getAuthor().get().asMember(message.getGuild().block().getId()).block();
     Color color = sender.getColor().block();
-    String roleHex = String.format("#%02X%02X%02X", color.getRed(), color.getBlue(), color.getGreen());
+
+    // fully qualified name to avoid name collisions with discord4j Color
+    java.awt.Color rankColor = new java.awt.Color(color.getRed(), color.getGreen(), color.getBlue());
```

Comment:
The only changes pertaining to the colored names feature are here, the new line 30, and the new line 43. The rest of the changes pertain to the text formatting feature.